lida/datamodel.py

```diff
@@ -24,21 +26,35 @@ class CompletionResult:
 @dataclass
 class UploadUrl:
     """Response from a text generation"""
+
     url: str
 
 
 @dataclass
 class Goal:
     """A visualization goal"""
+
     index: int
     question: str
     visualization: str
     rationale: str
 
+    def _repr_markdown_(self):
```

Comment:
Thanks for the _repr_'s. 
I learned a few new things here @rgbkrk !!

Reply:
Always happy to sprinkle on some ipython goodness. 😄